cli/cmd/release/prepare/README.md

```diff
@@ -0,0 +1,29 @@
+# prepare
+
+The `prepare` command is responsible for generating the checklists that make up the release process.
```

Comment:
This shouldn't say "checklists" right? Maybe it should say "pull requests" ?

Reply:
Good catch, updated 😎